samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -79,6 +79,18 @@ Recovery mode
 In the provisioned state, you can enter the recovery mode for a limited time.
 This mode allows the Android device to recover a lost provisioning key from the locator tag.
 
+DULT Motion detector active mode
+--------------------------------
+
+In the provisioned state, the accessory can enter the DULT motion detector active mode.
+It happens when the accessory is separated from the owner for a randomized amount of time between :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_SEPARATED_UT_TIMEOUT_PERIOD_MIN` and :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_SEPARATED_UT_TIMEOUT_PERIOD_MAX`.
```

Comment:
It makes more sense to describe such specifics in the detailed guide (e.g. integration) and use more vague values in the sample description (aka. sufficient time). Let's keep the sample description very high-level. If the user wants to dig deeper, they can use more details documentation.